gradle/maven.gradle

```diff
@@ -29,6 +29,9 @@ plugins.withId('maven') {
                             }
                         }
                     }
+                    else if (publishUrl == "localFileRepo") {
+                        repository(url : "file://${buildDir}/repo/snapshots")
```

Comment:
nit: please change this to `repository/snapshots`

Reply:
Done